jira/resources.py

```diff
@@ -608,14 +608,14 @@ def __init__(self, options, session, raw=None):
         if raw:
             self._parse_raw(raw)
 
-    def update(self, fields=None, async_=None, jira=None, body="", visibility=None):
+    def update(self, fields=None, async_=None, jira=None, body="", visibility=None, notify=True):
         """Update a comment"""
```

Comment:
@ronniel1 , could you please also extend the documentation of this method to describe the parameters